learn/core_concepts/primary_key.md

```diff
@@ -232,3 +232,31 @@ This happens when your document id does not have the correct [format](#formattin
     "finishedAt": "2021-12-30T11:28:59.084803Z"
 }
 ```
+
+### `index_primary_key_multiple_candidates_found`
```

Comment:
Should this follow directly the other new error, since they both have to do with primary key inference?

Reply:
It would be better to list these errors in alphabetical order as we do in `error_codes.md`

Reply:
I disagree.

Alphabetical order makes sense for a reference file like `error_codes.md`, where people will be searching for a specific error. Being alphabetical helps them find that error more quickly.

Unlike `error_codes.md`, this page is intended to be read sequentially. Because of that, these errors should be grouped by topic/conceptually.

Reply:
In this case, it seems like alphabetical order puts the errors in an intuitive order anyway, so further changes may not be required. Just something to think about in the future.